src/main/java/by/itechart/api/util/annotation/ApiPageable.java

```diff
@@ -0,0 +1,20 @@
+package by.itechart.api.util.annotation;
+
+import io.swagger.annotations.ApiImplicitParam;
+import io.swagger.annotations.ApiImplicitParams;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target({ElementType.FIELD, ElementType.ANNOTATION_TYPE, ElementType.TYPE, ElementType.METHOD})
+@Retention(RetentionPolicy.RUNTIME)
+@ApiImplicitParams({
+        @ApiImplicitParam(name = "page", dataType = "integer", paramType = "query", defaultValue = "0", value = "Results page you want to retrieve (0..N)"),
+        @ApiImplicitParam(name = "size", dataType = "integer", paramType = "query", defaultValue = "10", value = "Number of records per page."),
+        @ApiImplicitParam(name = "sort", allowMultiple = true, dataType = "string", paramType = "query", value = "Sorting criteria in the format: property(,asc|desc). "
```

Comment:
Good description in `value` :+1: 
You can also list properties available for sorting and add default value since we are using `fistName` as default filed for sorting

Reply:
Thanks! :3
Actually, I don't know the right approach to how to list properties for sorting, but please check my new implementation.